README.md

```diff
@@ -1,4 +1,4 @@
-# Supercronic #
+# ![](https://raw.github.com/aptible/straptible/master/lib/straptible/rails/templates/public.api/icon-60px.png) Supercronic #
```

Comment:
Adding a logo is a quick way to increase awareness about Aptible without adding unnecessary text. We might consider a refreshed one with the "Aptible" mark that links.